webapp/lib/view.dart

```diff
@@ -203,6 +203,7 @@ class ProjectSelectorView {
       var data = new controller.ProjectSelectionData()..isProjectSelection = true;
       controller.command(controller.UIAction.needsReplyDataUpdated, data);
       controller.command(controller.UIAction.systemEventsDataUpdated, data);
+      ChartFiltersView().filterChartsByPeriod(_);
```

Comment:
I see that `filterChartsByPeriod` doesn't actually use whatever parameter is passed into it, so I'd just remove it. It's also not good practice to pass unnamed parameters forwards - if you're using them, they should be named, but it this case you don't need it.

Reply:
Noted. Fixed